storybooks/storybooks.github.io
# Fix full page refresh problem

Issue: #41 

## What I did

Updated to use react-router links

## How to test
```
yarn develop
```
Click on links around the site and observe that page does not refresh.

```
yarn storybook
```
Stories still work properly because Gatsby uses an old version of react-router.
